MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
- [ ] Nice for adding the license :+1: But please customize it by adding your name instead of **[YOUR NAME]** cause it is very important to have the author name :smile: